### ISSUE TITLE
Add the id of the job that failed and check if there's no alternative handling this error

### ISSUE THREAD

**Issue by huineng** [IBM/diem#54]
```
trace
[
 "@at $job.stop (jobStop)",
 "@at $job.logger (jobLogger) - handleMail",
 "@at $handle.mail (handleMail)",
 "@at $handle.mail (prepareMail)",
 "@at $mail.notifications (sendMail)",
 "@at $mailhanlder (newMail)",
 "@at $mail (sendMail)"
]
body: [object Object]
transid: 54d0c3ba-e2e3-0902-6c98-9840096ac3cf
caller: @at $job.stop (jobStop)
log: $job.start.handler (saveDoc): error
```

**Comment by huineng** [comment]
cause when the owner of the document also add his/her name but with some uppercase ..as we only filter on lowercase, the email is not filtered out

https://github.com/IBM/diem/blob/ecc9c1b3a8142020540e06dc68a7f535c6380bff/applications/diem-core/src/server/routes/mail/mail.handler.ts#L78-L80

Need to add lowercase filter

**Comment by huineng** [comment]
alse here 
https://github.com/IBM/diem/blob/ecc9c1b3a8142020540e06dc68a7f535c6380bff/applications/diem-core/src/server/routes/mail/handle.mail.ts#L143-L147

**Comment by huineng** [comment]
solved in 5ad5777a0ad8483f01c3ec87bf68184652ad0767
